filter_plugins/opg_custom.py

```diff
@@ -113,6 +113,24 @@ def find_rds_instance(rds_list, db_name):
     return db_info
 
 
+def merge_config_dictionaries(*dicts):
+    """
+    Merges n dictionaries of configuration data
+    :param dicts:
+    :return:
+    """
+    res_dict = {}
+
+    if len(dicts) <= 0:
+        return res_dict
+    elif len(dicts) == 1:
+        return dicts[0]
```

Comment:
nothing prevents you from calling this ```merge_config_dictionaries('x')```
and returning ```'x'``` here.

Reply:
you may want to check that *dicts is a single python dict, or a list containing python dicts.